drivers/SmartThings/zwave-thermostat/fingerprints.yml

```diff
@@ -41,7 +41,7 @@ zwaveManufacturer:
     productType: 0x6401
     productId: 0x0107
     deviceProfileName: base-thermostat
-  - id: "0098/6501/000C"
+  - id: "0098/6501/000C/000B"
```

Comment:
This is just changing the ID.  You will need to duplicate the entire entry (everything between this `-` entry and  the next one), but change the `productID` for the copy to use `000B` instead of C